nanohttp/tests/test_exceptions.py

```diff
@@ -109,3 +109,15 @@ def index(self):
     with pytest.raises(E):
         Given(Root(), configuration='debug: false')
 
+
+def test_http_no_content():
+    class Root(Controller):
+        @action
+        def index(self):
+            context.response_headers['x-app-extra'] = 'yep'
+            raise HTTPNoContent('Nothing to show')
+
+    with Given(Root()):
+        assert status == '204 Nothing to show'
+        assert response.body is None
+        assert response.headers['x-app-extra'] == 'yep'
```

Comment:
please Add a blank line.

Reply:
Where (exactly) missed?

Reply:
We add a blank line at the end of the file.

Reply:
Really weird situation with text editor, im still confused! but here is a blank line.

Reply:
@pylover all text editors shows new line `\n` as a blank line, except vim. and PEP8 linters shows warning if have two new line `\n\n` at end of file.

Reply:
@meyt Having one `\n` is ok.
Vim adds a `\n`  at the end of file by default, unlike Pycharm.

Reply:
@ArashFatahzade Yes, but nanohttp files have double new line (\n\n) at end of file. linters says its not ok.

Reply:
@meyt In some cases, we don't follow PEP8, like this case(No longer) or breaking function call. But now, Having only one `\n` is ok.

Reply:
seems longer discussion for here, @ArashFatahzade i've added new line as what your team follows, @pylover please review and merge changes. 
@ArashFatahzade also it will be good to specify linter that used on this project, and which rules should suppressed.

Reply:
> I think the github is not showing the last empty line.

@pylover there it is

https://github.com/Carrene/nanohttp/pull/137/commits/314eaebd15c1c9fc301a7bb61f9d41ec823f661b

![image](https://user-images.githubusercontent.com/10482278/53684003-2aa03880-3d1d-11e9-8d90-0ae47e9762d0.png)